assets/js/modules/adsense/components/dashboard/AdSenseDashboardWidget.js

```diff
@@ -182,17 +183,6 @@ class AdSenseDashboardWidget extends Component {
 								wrapperClass
 							) }>
 								<ModuleSettingsWarning slug="adsense" context="module-dashboard" />
-								<Layout
-									header
-									title={ __( 'Estimated earnings', 'google-site-kit' ) }
-									headerCtaLabel={ __( 'Advanced Settings', 'google-site-kit' ) }
-									headerCtaLink={ homepage }
-								>
-									<AdSenseEstimateEarningsWidget
-										handleDataError={ this.handleDataError }
-										handleDataSuccess={ this.handleDataSuccess }
-									/>
```

Comment:
While this component and its usage should be removed, the removal of passing the `handleDataError` and `handleDataSuccess` props still causes the same problem here that I mentioned in the previous PR: If the AdSense API returns only 0 numbers, this `handleDataError` needs to be triggered, otherwise `handleDataSuccess` needs to be triggered. I assume because this was no longer happening, this is why you removed `loading` above, which shouldn't be the case though; we need to maintain this behavior. What you should see if you have only 0s in your AdSense data is a prompt informing you that you might need to wait to get impressions instead of the regular screen UI - this is what this logic is used for.

Since the `handleDataSuccess` and `handleDataError` props are automatically handled in the `withData` HOC that is also used for an AdSense API request in the `AdSensePerformanceWidget`, you should be able to pass the props to that component below.